src/admin/containers/email.js

```diff
@@ -83,14 +92,14 @@ const BouncedEmails = ({ bounced }) =>
   </div>
 
 const map_tab =
-  { SEND_SUB_REMINDER: email_list
-  , SEND_NEWSLETTER: email_list
-  , SEND_NEWSLETTER_REMINDER: email_list
-  , COMPOSE_CUSTOM: props => <CustomEmailForm submit={props.submit_custom_email} members={props.custom_emails.members}/>
-  , GET_BOUNCED: BouncedEmails
+  { [SEND_SUB_REMINDER]: email_list
+  , [SEND_NEWSLETTER]: email_list
+  , [SEND_NEWSLETTER_REMINDER]: email_list
+  , [SUB_DUE_TAB]: props => <SubDueSection fetch_sub_due={props.send_subscription_due_email} {...props} component={email_list} checker={!isEmpty(props.emails)}/>
```

Comment:
I'm not sure about these inline component functions. Any reason not to pull them out?